docs/en/observability/monitor-status-alert.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[monitor-status-alert]]
+= Create a monitor status alert
+
+Within the {uptime-app}, create a *Monitor Status* alert to receive notifications
+based on errors and outages. 
+
+. In the side navigation, expand *Observability*, and then click *Uptime*.
+. On the *Overview* page, click *Alerts*, and then select *Create alert*.
+. Select *Monitor status alert*.
+
+[TIP]
+===============================
+If you already have a query in the overview page search bar, it's populated here.
+===============================
+
+[[status-alert-conditions]]
+== Conditions
+
+You can specify the following thresholds for your alert.
+
+|=== 
+
+| *Status check* | Receive alerts when a monitor goes down a specified number of
+times within a time range (seconds, minutes, hours, or days).
+
+| *Availability* | Receive alerts when a monitor goes below a specified availability
+threshold within a time range (days, weeks, months, or years).
+
+|=== 
+
+Let's create an alert for any monitor that shows `Down` more than three times in 10 minutes.
+So you are not flooded with alert notifications, you only want to be notified
+every 30 minutes.
```

Comment:
This sentence was tough for me to digest. Comma splice? Maybe something like

```
We'll choose to only be notified every 30 minutes so that we're not flooded with alert notifications.
```

Reply:
Much better. Thanks.